sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -483,7 +482,9 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
    * @since 2.0.0
    */
   def countMinSketch(col: Column, depth: Int, width: Int, seed: Int): CountMinSketch = {
-    countMinSketch(col, CountMinSketch.create(depth, width, seed))
+    val eps = 2.0 / width
+    val confidence = 1 - 1 / Math.pow(2, depth)
```

Comment:
Why need the two? `eps = 2.0 / width` and `confidence = 1 - 1 / Math.pow(2, depth)`?

Reply:
? what do you mean?

Reply:
this is used to convert `depth & width` to `eps & confidence`

https://github.com/apache/spark/blob/b0576fff9b72880cd81a9d22c044dec329bc67d0/common/sketch/src/main/java/org/apache/spark/util/sketch/CountMinSketchImpl.java#L44-L45

Reply:
`CountMinSketchAgg` doesn't have a constructor with `depth & width`

Reply:
I got it.